lib/escalas/assist/assist_screen.dart

```diff
@@ -148,25 +146,19 @@ class _AssistScreenState extends State<AssistScreen> {
         padding: const EdgeInsets.all(30.0),
         child: _questionIndex < _questions.length
             ? Assist(
+                sizeQuestionnaire: _questions.length - 1,
```

Comment:
pq -1?

Reply:
Por que o tamanho do questionário tem a primeira questão contada junto, mas ela não deve ser contada no total

Reply:
Pq não?

Reply:
o index começa no 0 e dai vai aparecer "0 de 10 questoes", mas ai quando chegar na questão 9 de 10 ele vai pra tela final e não para a questão 10 de 10. Se não for aparecer ai o -1 vai ter que aparecer na outra pagina o +1 do index